### PR TITLE
Bump to Python3

### PR DESCRIPTION
**Pull request by kumarde**
This PR will bump iot-inspector to Python3, in light of the imminent demise of Python 2.7.

This removes the dependency on
- scapy_ssl_tls
- netdisco_wrapper

As well as appropriately casting types and general Py3 syntactic sugar.

### CODE REVIEW

**Comment by kumarde** [comment]
Tested on Linux, OSX. Think it's ready to merge if you want to take a look @crazyideas21!

**Comment by crazyideas21** [comment]
Nice. Thank you! Let me take a look tonight.

File: v2-src/packet_processor.py [hunk 1]
```diff
@@ -189,8 +189,8 @@ def _process_dns(self, pkt):
             return
 
         # Remove trailing dot from domain
-        if domain.endswith('.'):
-            domain = domain[0:-1]
+        if domain[-1] == '.':
+            main = domain[0:-1]
```
Comment (crazyideas21) [inline v2-src/packet_processor.py]: @kumarde mmm unused variable. typo?

**Comment by crazyideas21** [comment]
@kumarde I see that you've removed files in the `dist` directory. 

In my original workflow, every time I wanted to push a new macOS binary, I would run `deploy.sh`, which packages the macOS binary into the `dist` directory, and then I will commit and push the changes to github. The [download link](https://iot-inspector.princeton.edu/blog/post/getting-started/#i-class-fab-fa-apple-i-macos) points to that packaged binary from github. Checking in binaries to github may not be a good practice; at least it's easy enough for me...

Now that you are removing `dist`,  what's your proposed solution of deploying new macOS binaries? 

I'm going to merge your PR regardless; just wanted to hear your thoughts on software updates and deployment.

**Comment by kumarde** [comment]
Oh, great catch on `dist`. I just always thought the binaries were distributed out of band (i.e., not on GitHub.) 

I'm happy to bring that back in – in general, I thought committing compiled unversioned binaries to GitHub could lead to weird build mismatches in the dev process.

**Comment by crazyideas21** [comment]
> I'm happy to bring that back in – in general, I thought committing compiled unversioned binaries to GitHub could lead to weird build mismatches in the dev process.

I didn't realize that'd be an issue. Feel free to keep `dist/` removed.

I'll merge the PR once you fix https://github.com/noise-lab/iot-inspector-client/pull/30#discussion_r346937392

**Comment by gunesacar** [comment]
@crazyideas21 Re: removal of `dist/`; Github has a Release interface that allows you to upload and version binaries: https://help.github.com/en/github/administering-a-repository/creating-releases

**Comment by crazyideas21** [comment]
Thanks for the pointers @gunesacar!

File: v2-src/packet_processor.py (hunk 1 above)
Comment (kumarde) [reply v2-src/packet_processor.py]: Haha, whoops. It should be DOmain, not sure how that happened.

**Comment by crazyideas21** [comment]
I'm putting `dist` back to the repo for backward compatibility, because removing `dist` caused [this issue](https://github.com/noise-lab/iot-inspector-client/issues/35) for [this reason](https://github.com/noise-lab/iot-inspector-client/commit/decfac1535b3b471249db88ca6a48bb6ee3f7b9c).

**Comment by kumarde** [comment]
Whoops, thanks for catching that. Didn't realize it would break macOS
deployment!

**Pull request merged**